pkg/utils/utils.go

```diff
@@ -131,6 +131,9 @@ func SyncNodeState(newState *sriovnetworkv1.SriovNetworkNodeState) error {
 }
 
 func needUpdate(iface *sriovnetworkv1.Interface, ifaceStatus *sriovnetworkv1.InterfaceExt) bool {
+	if ifaceStatus.Vendor == "15b3" && ifaceStatus.TotalVfs < iface.NumVfs {
```

Comment:
why do we need this change?

Reply:
if the new numOfVfs > old totalVfs then the generic plugin will fail